test/smoke/default/compose.rb

```diff
@@ -0,0 +1,6 @@
+# # encoding: utf-8
+
+# Inspec test for recipe osl-docker::compose
+
+# The Inspec reference, with examples and extensive documentation, can be
+# found at http://inspec.io/docs/reference/resources/
```

Comment:
Are we planning on adding Inspec tests in the future?

Reply:
Yes, but this won't be the correct location. I'll remove this file.